src/devices/Mhz19b/Mhz19b.cs

```diff
@@ -0,0 +1,256 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.IO.Ports;
+using System.Text;
+using System.Threading;
+using UnitsNet;
+
+namespace Iot.Device.Mhz19b
+{
+    /// <summary>
+    /// MH-Z19B CO2 concentration sensor binding
+    /// </summary>
+    public sealed class Mhz19b : IDisposable
+    {
+        private enum Command : byte
```

Comment:
those should be defined in the bottom of the class (style guidelines)

Reply:
Done and will be remembered in future